pyproject.toml

```diff
@@ -1,7 +1,79 @@
+# See https://packaging.python.org/en/latest/specifications/declaring-project-metadata
+
+[tool.poetry]
+name = "dsp-tools"
+version = "1.22.2"
+description = "DSP-TOOLS is a Python package with a command line interface that helps you interact with a DaSCH service platform (DSP) server."
+authors = ["DaSCH - Swiss National Data and Service Center for the Humanities <info@dasch.swiss>"]
+readme = "docs/index.md"
+license = "GPL-3.0-only"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: OS Independent",
+]
+documentation = "https://docs.dasch.swiss/latest/DSP-TOOLS/"
+homepage = "https://www.dasch.swiss/"
+repository = "https://github.com/dasch-swiss/dsp-tools"
+include = [
+    "src/dsp_tools/schemas/*",
+    "src/dsp_tools/docker/*"
+]
+exclude = [
+    "src/dsp_tools/import_scripts/*"
+]
+
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.12"
+jsonpath-ng = "^1.5.3"
+argparse = "^1.4.0"
+lxml = "^4.9.2"
+requests = "^2.28.1"
+jsonschema = "^4.17.3"
+pystrict = "^1.3"
+openpyxl = "^3.0.10"
+networkx = "^2.8.8"
+pandas = "^1.5.2"
+xlrd = ">=1.0.0"            # without this, pandas raises an ImportError: Missing optional dependency 'xlrd'
```

Comment:
I'm not yet familiar with poetry, but in most build tools (including pip and pipenv) you can specify optional dependencies directly on the package requiring them. Maybe worth having a look if that's possible here too?

Reply:
I tried to find a more elegant solution, but to no avail. According to https://stackoverflow.com/a/71387157/14414188, the following should work: 
```
pandas = {extras = ["excel"], version = "^1.5.2"}
```
But unfortunately, it does NOT work. So I guess I will just leave it as it is.